src/components/map/MapService.js

```diff
@@ -860,13 +861,17 @@ goog.require('ga_urlutils_service');
           return urls;
         };
 
-        var getWmtsGetTileTpl = function(tpl, layer, time, tileMatrixSet,
-            format, use2dTpl) {
-          var dfltTpl = use2dTpl ? wmtsGetTileUrlTemplate :
-              wmtsMapProxyGetTileUrlTemplate;
-          var url = (tpl || dfltTpl)
-              .replace('{Layer}', layer)
-              .replace('{Format}', format);
+        var getWmtsGetTileTpl = function(layer, time, tileMatrixSet,
+            format, use2dTpl, useNativeTpl) {
+          var tpl;
+          if (use2dTpl) {
+              tpl = wmtsGetTileUrlTemplate;
```

Comment:
Same comment for the multiple WMTS templating, since we know we need the 2d template we could transform `wmtsNativeGetTileUrlTemplate`